test/controllers/endorseCompanyClosureCertificate.controller.test.ts

```diff
@@ -56,4 +68,48 @@ describe('EndorseCompanyClosureCertificateController', () => {
       assert.isTrue(htmlAssertHelper.hasText('span#applicantName', 'John Smith'))
     })
   })
+
+  describe('POST - ensure form submission is handled correctly', () => {
+    it('should redirect successfully if validator returns no errors', async () => {
+      const testObject: EndorseCertificateFormModel = {confirmation: 'understood'}
```

Comment:
Objects should always be created using fixtures. Create a new fixture called `generateEndorseCertificateFormModel` as per other controllers. This fixture should be used any time we are creating this type of object